server/app/views/dev/DatabaseSeedView.java

```diff
@@ -223,6 +224,17 @@ private SectionTag createEmailSection() {
                 .withTarget("_blank"));
   }
 
+  private SectionTag createAddressToolsSection() {
+    return section()
+        .with(h2("Address Tools").withClass("text-2xl"))
+        .with(p("View address lookup and eligibility results"))
+        .withClasses("flex", "flex-col", "gap-4", "border", "border-black", "p-4")
+        .with(
+            createLink(
+                "\uD83D\uDDFA\uFE0F Go to address tools",
```

Comment:
**question:** What is `\uD83D\uDDFA\uFE0F` for?

Reply:
Map emoji 🗺️

;)

Reply:
Haha love it! Can you add a comment :)

Reply:
Already did